docs/source/changelog.rst

```diff
@@ -12,6 +12,54 @@ For more detailed information, see
      Use ``pip install notebook --upgrade`` or ``conda upgrade notebook`` to
      upgrade to the latest release.
 
+.. _release-5.0.0:
+
+Notebook version 5.0
+====================
+
+This is the first major release of the Notebook Application and Server after
+the "Big Split" of IPython into IPython and Jupyter.
+
+A reminder that most of the development cycles are now on JupyterLab and that
+we encourage users to try it out, and send feedback.
```

Comment:
I'd like to avoid "which is nearing a beta release" as the "what's new" will likely persist after JupyterLab becomes beta. So I'd like something a bit more intemporal. I've rephrased.